aws-lc-sys/builder/cc/aarch64-apple-darwin.toml

```diff
@@ -0,0 +1,314 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Not really sure this is worth the toml/serde dependency... Why not just generate a rust file and `include!` it? You can even have it create a `struct` value as well:

```rust
static CRYPTO_LIB: Library = Library {
    name: "crypto",
    flags: &[],
    sources: &[...],
};
```

If you want better scoping your could also use the `#[path]` attribute on a module:

```rust
#[path = "./aarch64-apple-darwin.rs"]
mod libraries;
```

Reply:
One reason to not generate this information in Rust syntax is that AWS-LC source file information could be useful beyond just building our (Rust-specific) sys-crates. For example, we could potentially generate TOML files like these as part of an AWS-LC release for use by other build processes or static/dynamic-analysis.

That said, updating the shell script to generate Rust-syntax should be fairly easy, and elimination of the toml/serde dependencies could reduce "maintenance costs" long term. I'll look into it.

Reply:
> For example, we could potentially generate TOML files like these as part of an AWS-LC release for use by other build processes or static/dynamic-analysis.

That should still be possible with the rust files, too. Even if you just parse it with `syn` in your analysis script (and maybe that could produce the toml files). I just would prefer reducing the number of dependencies here, since that usually causes the most problems for maintenance.